elements/lisk-client/cypress/plugins/index.js

```diff
@@ -0,0 +1,35 @@
+/// <reference types="cypress" />
```

Comment:
`.ts` file extension? https://docs.cypress.io/guides/tooling/typescript-support.html#Types-for-plugins

Reply:
These are autogenerated file from cypress, kept as skeleton to be useful later.